jaxrs-clients/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     shadow project(':error-handling')
     shadow project(':service-config')
     shadow project(':ssl-config')
+    shadow project(':tracing-okhttp3')
```

Comment:
are we able to programmatically tie this to the shadowJar dependencies configuration?